style.css

```diff
@@ -57,4 +58,12 @@
     text-align:center;
     font-weight: bolder;
     margin-top:50px;
+}
+
+
+@media only screen and (min-width:300px) and (max-width:600px){
```

Comment:
I think there is no need for this media query.  
you can add media query for big screens(when image width reaches it's max-width) and make the image and timeline be side by side.